spyder/plugins/outlineexplorer/widgets.py

```diff
@@ -295,6 +295,9 @@ def set_current_editor(self, editor, update):
 
     def file_renamed(self, editor, new_filename):
         """File was renamed, updating outline explorer tree"""
+        if editor is None:
+            # We can't do anything

```

Comment:
Please change this comment to:

    # This is needed when we can't find an editor to attach
    # the outline explorer to.
    # Fixes issue 8926

Reply:
# Fix issue 8813